src/pages/postgraphile/usage-library.md

```diff
@@ -63,7 +63,7 @@ The `postgraphile` middleware factory function takes three arguments, all of whi
   - `setofFunctionsContainNulls`: If none of your `RETURNS SETOF compound_type` functions mix NULLs with the results then you may set this true to reduce the nullables in the GraphQL schema.
   - `classicIds`: Enables classic ids for Relay support. Instead of using the field name `nodeId` for globally unique ids, PostGraphile will instead use the field name `id` for its globally unique ids. This means that table `id` columns will also get renamed to `rowId`.
   - `disableDefaultMutations`: Setting this to `true` will prevent the creation of the default mutation types & fields. Database mutation will only be possible through Postgres functions.
-  - `ignoreRBAC`: Set false (recommended) to exclude fields, queries and mutations that the user isn't permitted to access from the generated GraphQL schema; set this option true to skip these checks and create GraphQL fields and types for everything. The default is `true`, in v5 the default will change to `false`.
+  - `ignoreRBAC`: Set false (recommended) to exclude fields, queries and mutations that the user account you connect to PostgreSQL with (from your connection string)  isn't permitted to access from the generated GraphQL schema; set this option true to skip these checks and create GraphQL fields and types for everything. The default is `true`, in v5 the default will change to `false`.
```

Comment:
This line is auto-generated from the source, so will be lost; please make the change here instead:

https://github.com/graphile/postgraphile/blob/3d35995bce11526b6ab2754cd79c27af8ced16a0/src/interfaces.ts#L88-L91